src/main/java/ch/njol/skript/command/ScriptCommand.java

```diff
@@ -145,6 +148,25 @@ public ScriptCommand(
 			this.permissionMessage = permissionMessage;
 		}
 
+		if (prefix != null) {
+			for (char c : prefix.toCharArray()) {
+				if (Character.isWhitespace(c)) {
+					Skript.warning("command /" + name + " has a whitespace in its prefix. Defaulting to '" + ScriptCommand.DEFAULT_PREFIX + "'.");
```

Comment:
Prob don't need to mention ScriptCommand since we're in that class now :) (same for other warning)
```suggestion
					Skript.warning("command /" + name + " has a whitespace in its prefix. Defaulting to '" + DEFAULT_PREFIX + "'.");
```